### PR TITLE
Add 'use strict' to files

### PR DESCRIPTION
**Pull request by kjdelisle**
connected to https://github.com/strongloop-internal/scrum-nodeops/issues/1596

### CODE REVIEW

**Comment by kjdelisle** [comment]
@sam-github PTAL

**Comment by sam-github** [comment]
commit message format!

File: index.js [hunk 1]
```diff
@@ -52,4 +54,3 @@ function buildCommit(commitOpts) {
 DeployReceiver.prototype.buildCommit = buildCommit;
 
 module.exports = DeployReceiver;
-
```
Comment (sam-github) [inline index.js]: unrelated whitespace change?

**Comment by sam-github** [comment]
other than msg format, LGTM

File: index.js (hunk 1 above)
Comment (kjdelisle) [inline index.js]: I think my editor pruned the redundant whitespace here...

**Comment by kjdelisle** [comment]
@slnode test please
